aiven/client/cli.py

```diff
@@ -1136,6 +1136,140 @@ def service_acl_list(self):
         self.print_response(service.get("acl", []), json=self.args.json,
                             table_layout=layout)
 
+    @arg.project
+    @arg.service_name
+    def service_connector_available(self):
+        """List available Kafka connectors"""
+        project_name = self.get_project()
+
+        response = self.client.get_available_kafka_connectors(project_name, self.args.name)
+
+        self.print_response(response)
+
+    @arg.project
+    @arg.service_name
+    def service_connector_list(self):
+        """List Kafka connectors"""
+        project_name = self.get_project()
+
+        response = self.client.list_kafka_connectors(project_name, self.args.name)
+
+        self.print_response(response)
+
+    @arg.project
+    @arg.service_name
+    @arg("connector", help="Connector name")
+    def service_connector_status(self):
+        """Get Kafka connector status"""
+        project_name = self.get_project()
+
+        response = self.client.get_kafka_connector_status(project_name, self.args.name, self.args.connector)
+
+        self.print_response(response)
+
+    @arg.project
+    @arg.service_name
+    @arg("connector", help="Connector name")
+    def service_connector_configuration(self):
+        """Get Kafka connector configuration"""
+        project_name = self.get_project()
+
+        response = self.client.get_kafka_connector_configuration(project_name, self.args.name, self.args.connector)
+
+        self.print_response(response)
+
+    def get_connector_config(self, path_or_string):
+        config = path_or_string
+        if os.path.isfile(config):
+            with open(config, 'r') as config_file:
```

Comment:
Prefer `"` over `'` unless there's a specific reason not to (e.g. because the string itself contains `"` characters.